src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -122,6 +122,12 @@ public void clickSaveInstructorButton(int instrNum) {
         waitForPageToLoad();
     }
 
+    public void clickDiscardChangesLink() {
```

Comment:
This function assumes that the page will only contain 1 button with class `btn-discard-changes`. Not sure if it's good going forward, might be fine for now. Can someone else comment on this as well?

Reply:
I notice that there are some discrepancies between the implementation of discarding changes in `instructorFeedbackedit.js` and `instructorCourseEdit.js`. Currently the class `btn-discard-changes` in `instructorFeedbackEdit` page is involved in giving the confirmation modal and restoring the original question setting if the user is exiting the process of editing the question. If you use the same class across the file(s), either you want to give them the same styling or you want them to behave in the same way. 
Since currently the buttons in charge of exiting editting instructors do not give confirmation modals, I feel that applying class `btn-discard-changes` is of no use in `instructorCourseEdit` for now. Moreover, this element is unique in the page i.e. there will only be one of such button appear on the page at any time, I feel it would be better if we attach the action specific to it using ID.
(And I feel that the current use of `btn-discard-changes` is confusing in `instructorFeedbackEdit` as the elements from that class have actions both from the class and their respective IDs attached to them)

Reply:
@wenmogu 

Would a better choice be just cancelling the new instructor pane without showing a confirmation modal if the user clicks on the `cancel` button?

Reply:
Yeah I think it would be better, as there is not much setting to be done to the instructors

Reply:
Made the change! 

The new button follows the style of the cancel button in instructorCourseEditPage now. Please review :smile: